AuthenticatorShared/UI/Vault/VaultItem/AuthenticatorKeyCapture/ManualEntryView.swift

```diff
@@ -35,6 +35,7 @@ struct ManualEntryView: View {
             )
         }
         .buttonStyle(.tertiary())
+        .accessibilityIdentifier("manualEntryAddCodeButton")
```

Comment:
🤔 I see some identifiers start with lowercase and other with uppercase, shouldn't we maintain an standard with either one way or the other?

Reply:
thanks for catching this @fedemkr ! I followed the pattern we follow in the password manager side and switched to all uppercase